Demo/src/main/java/com/paypal/android/usecase/CreateOrderUseCase.kt

```diff
@@ -42,6 +42,28 @@ class CreateOrderUseCase @Inject constructor(
 
                 orderRequest.put("payment_source", paymentSourceJSON)
             }
+            if (request.appSwitchWhenEligible) {
+
+                val nativeAppJSON = JSONObject()
+                    .put("app_url", APP_URL)
+
+                val experienceContextJSON = JSONObject()
+                    .put("return_url", SUCCESS_URL)
+                    .put("cancel_url", CANCEL_URL)
+                    .put("native-app", nativeAppJSON)
```

Comment:
```suggestion
                    .put("native_app", nativeAppJSON)
```
qq: Is this not an underscore as well?

Reply:
it is underscore, corrected,